Packs/Sixgill/Integrations/Sixgill/Sixgill.yml

```diff
@@ -0,0 +1,81 @@
+category: Data Enrichment & Threat Intelligence
+fromversion: 5.0.0
+commonfields:
+  id: Sixgill
+  version: -1
+configuration:
+- display: Sixgill API client ID
+  name: client_id
+  required: true
+  type: 0
+- display: Sixgill API client secret
+  name: client_secret
+  required: true
+  type: 4
+- display: Max number of incidents that can be fetched
+  name: maxIncidents
+  required: false
+  type: 0
+- display: Filter by alert template severity
+  name: severity
+  options:
+  - low
+  - med
+  - high
+  required: false
+  type: 15
+- display: Filter by alert threat level
+  name: threat_level
+  options:
+  - imminent
+  - emerging
+  required: false
+  type: 15
+- display: Filter by alert threat type
+  name: threat_type
```

Comment:
please explain the detailed description where the users should achieve threat_type, where he should navigate? copy from?